app/src/main/java/com/onegini/mobile/exampleapp/view/activity/PendingPushMessagesActivity.java

```diff
@@ -0,0 +1,175 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.onegini.mobile.exampleapp.view.activity;
+
+import java.util.Set;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.design.widget.BottomNavigationView;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.Toolbar;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.TextView;
+import android.widget.Toast;
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import com.onegini.mobile.exampleapp.OneginiSDK;
+import com.onegini.mobile.exampleapp.R;
+import com.onegini.mobile.exampleapp.adapter.PendingPushMessagesAdapter;
+import com.onegini.mobile.exampleapp.view.helper.NotificationMenuHelper;
+import com.onegini.mobile.sdk.android.handlers.OneginiPendingMobileAuthWithPushRequestsHandler;
+import com.onegini.mobile.sdk.android.handlers.error.OneginiPendingMobileAuthWithPushRequestError;
+import com.onegini.mobile.sdk.android.model.entity.OneginiMobileAuthWithPushRequest;
+import com.onegini.mobile.sdk.android.model.entity.UserProfile;
+
+public class PendingPushMessagesActivity extends AppCompatActivity {
+
+
+  @SuppressWarnings({ "unused", "WeakerAccess" })
+  @BindView(R.id.toolbar)
+  Toolbar toolbar;
+  @SuppressWarnings({ "unused", "WeakerAccess" })
+  @BindView(R.id.recycler_view)
+  RecyclerView recyclerView;
+  @BindView(R.id.pending_notifications_error)
+  TextView errorTextView;
+  @BindView(R.id.bottom_navigation)
+  BottomNavigationView bottomNavigationView;
+  @BindView(R.id.notifications_refresh_layout)
+  SwipeRefreshLayout swipeRefreshLayout;
+
+  private PendingPushMessagesAdapter adapter;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_pending_notifications);
+    ButterKnife.bind(this);
+    setupUi();
+  }
+
+  @Override
+  protected void onResume() {
+    super.onResume();
+    fetchPendingTransactions();
+  }
+
+  private void setupUi() {
+    setupActionBar();
+    setupListView();
+    setupNavigationBar();
+  }
+
+  private void setupListView() {
+    adapter = new PendingPushMessagesAdapter(this);
+    recyclerView.setAdapter(adapter);
+    recyclerView.setLayoutManager(new LinearLayoutManager(this));
+    swipeRefreshLayout.setOnRefreshListener(() -> fetchPendingTransactions());
```

Comment:
can be replaced with method reference

Reply:
I've finally found a way to make such warnings more visible in my IDE, hope it won't happen so often anymore :)